proposals/0029-cooperative-vector.md

```diff
@@ -647,7 +647,7 @@ typedef enum D3D12_LINEAR_ALGEBRA_DATATYPE {
   D3D12_LINEAR_ALGEBRA_DATATYPE_UINT32          =  5, // ComponentType::U32
   D3D12_LINEAR_ALGEBRA_DATATYPE_FLOAT16         =  7, // ComponentType::F16
   D3D12_LINEAR_ALGEBRA_DATATYPE_FLOAT32         =  8, // ComponentType::F32
-  D3D12_LINEAR_ALGEBRA_DATATYPE_INT8_T4_PACKED  = 16, // ComponentType::PackedS8x32
+  D3D12_LINEAR_ALGEBRA_DATATYPE_SINT8_T4_PACKED = 16, // ComponentType::PackedS8x32
```

Comment:
I thought this should be SINT8 to be locally consistent.   But was INT8 used to intentionally match the use in another context?

Reply:
Could revert this if _INT8_T4 is an important name to keep.  @anupamachandra, @damyanp thoughts?